fvtest/porttest/omrsockTest.cpp

```diff
@@ -304,6 +306,47 @@ TEST(PortSockTest, getaddrinfo_and_freeaddrinfo)
 	EXPECT_EQ(rc, 0);
 }
 
+/**
+ * Test functions involving IPv4 and IPv6 socket addresses.
+ *
+ * After creating the address, their ability to bind and listen will be tested.
+ * 
+ * Address families tested include IPv4 and IPv6.
+ *
+ * @note Errors such as failed function calls, failure to create socket addresses, failure 
+ * to set up bind, listen with the socket address, and so on, will be reported.
+ */
+TEST(PortSockTest, socket_address)
+{
+	OMRPORT_ACCESS_FROM_OMRPORT(portTestEnv->getPortLibrary());
+
+	OMRSockAddrStorage sockAddr;
+	uint16_t port = 4930;
+	uint8_t addr[4];
+	uint8_t addr6[16];
+
+	/* To Create a Socket Address with INADDR_ANY IPv4 address*/
+	uint32_t inaddrAny = OMRPORTLIB->sock_htonl(OMRPORTLIB, OMRSOCK_INADDR_ANY);
+	memcpy(addr, &inaddrAny, 4);
+	EXPECT_EQ(OMRPORTLIB->sock_sockaddr_init(OMRPORTLIB, OMRSOCK_AF_INET, addr, OMRPORTLIB->sock_htons(OMRPORTLIB, port),  &sockAddr), 0);
+
+	/* To Create a Socket Address with dotted-decimal IPv4 address. */
+	EXPECT_EQ(OMRPORTLIB->sock_inet_pton(OMRPORTLIB, OMRSOCK_AF_INET, "127.0.0.1", addr), 0);
+	EXPECT_EQ(OMRPORTLIB->sock_sockaddr_init(OMRPORTLIB, OMRSOCK_AF_INET, addr, OMRPORTLIB->sock_htons(OMRPORTLIB, port),  &sockAddr), 0);
+
+	/* To Create a Socket Address with the IPv6 address. */
+	EXPECT_EQ(OMRPORTLIB->sock_inet_pton(OMRPORTLIB, OMRSOCK_AF_INET6, "::1", addr6), 0);
+	EXPECT_EQ(OMRPORTLIB->sock_sockaddr_init6(OMRPORTLIB, OMRSOCK_AF_INET6, addr6, OMRPORTLIB->sock_htons(OMRPORTLIB, port), 0, 0, &sockAddr), 0);
+
+	/* To Create a Socket Address with the in6addr_any IPv6 address. */
+	EXPECT_EQ(OMRPORTLIB->sock_inet_pton(OMRPORTLIB, OMRSOCK_AF_INET6, "::0", addr6), 0);
+	EXPECT_EQ(OMRPORTLIB->sock_sockaddr_init6(OMRPORTLIB, OMRSOCK_AF_INET6, addr6, OMRPORTLIB->sock_htons(OMRPORTLIB, port), 0, 0, &sockAddr), 0);
+
+	/* To Create a Socket Address with IPv4-mapped IPv6 address. */
+	EXPECT_EQ(OMRPORTLIB->sock_sockaddr_init6(OMRPORTLIB, OMRSOCK_AF_INET, addr, OMRPORTLIB->sock_htons(OMRPORTLIB, port), 0, 0, &sockAddr), 0);
+}
+
```

Comment:
extra newline

Reply:
This one is still here

Reply:
Sorry I must have missed it somehow; I made the change just now.